docker-compose.yml

```diff
@@ -17,6 +18,23 @@ services:
       - "${PWD}:/app"
     command: gunicorn pyslackersweb:app_factory --access-logfile - --bind=0.0.0.0:8000 --worker-class=aiohttp.GunicornWebWorker --reload
 
+  manage:
```

Comment:
This can be run by the main app’s command, probably better to do that than duplicate.